packages/tpetra/core/test/PerformanceCGSolve/cg_solve_file.cpp

```diff
@@ -83,16 +82,6 @@ bool cg_solve (Teuchos::RCP<CrsMatrix> A, Teuchos::RCP<Vector> b, Teuchos::RCP<V
   p = Tpetra::createVector<ScalarType>(A->getRangeMap());
   Ap = Tpetra::createVector<ScalarType>(A->getRangeMap());
 
-  {
-    TimeMonitor t(*TimeMonitor::getNewTimer(replaceValsName));
-    int length = r->getLocalLength();
-    for(int i = 0;i<length;i++) {
-      x->replaceLocalValue(i,0);
-      r->replaceLocalValue(i,1);
-      Ap->replaceLocalValue(i,1);
-    }
-  }
-
```

Comment:
As another note, this loop was taking up 90% of solve time with --size=200 and on one GPU of Summit.